cli/azd/pkg/auth/memory_cache.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package auth
+
+import (
+	"bytes"
+	"log"
+
+	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/cache"
+)
+
+var _ cache.ExportReplace = &memoryCache{}
```

Comment:
More questions about these lines.  I think I'm convinced that this will just ensure that these types correctly implement the specified interface.

Reply:
Yup - I'll go ahead and remove the ones that I can anyway.  It ends up that given how I develop, I put this line in first so I get the compiler errors to tell me how to go and develop the type to implement the interface I know I need.